framework/src/geomsearch/NearestNodeLocator.C

```diff
@@ -212,6 +223,82 @@ NearestNodeLocator::nearestNode(dof_id_type node_id)
   return _nearest_node_info[node_id]._nearest_node;
 }
 
+void
+NearestNodeLocator::updatePatch(std::vector<dof_id_type> & recheck_slave_nodes)
+{
+  Moose::perf_log.push("NearestNodeLocator::updatePatch()", "Execution");
+
+  std::vector<dof_id_type> trial_master_nodes;
+
+  // Build a bounding box.  No reason to consider nodes outside of our inflated BB
+  UniquePtr<BoundingBox> my_inflated_box = NULL;
+
+  const std::vector<Real> & inflation = _mesh.getGhostedBoundaryInflation();
+
+  // This means there was a user specified inflation... so we can build a BB
+  if (inflation.size() > 0)
+  {
+    BoundingBox my_box = MeshTools::create_local_bounding_box(_mesh);
+
+    Point distance;
+    for (unsigned int i = 0; i < inflation.size(); ++i)
+      distance(i) = inflation[i];
+
+    my_inflated_box =
+        libmesh_make_unique<BoundingBox>(my_box.first - distance, my_box.second + distance);
+  }
+
+  // Data structures to hold the Nodal Boundary conditions
```

Comment:
"Nodal Boundary conditions" -> "boundary nodes" ?

Reply:
Thanks for catching this. I will correct it.